src/input/src/styles/input.cssr.ts

```diff
@@ -305,6 +305,9 @@ export default c([
       justify-content: center;
       color: var(--suffix-text-color);
     `, [
+      cB('base-clear--no-pointer', {
```

Comment:
I think you should not write the code in `n-input`, because in some cases user may want the icon to respond to click events.

Reply:
The code should be removed.

Reply:
https://github.com/TuSimple/naive-ui/pull/292/files

now should be ok, thanks